app/api/dao/user.py

```diff
@@ -142,13 +143,14 @@ def get_user_by_username(username):
         return UserModel.find_by_username(username)
 
     @staticmethod
-    def list_users(user_id, search_query="", is_verified=None):
+    def list_users(user_id, search_query='', is_verified=None, sort_order=''):
         """ Retrieves a list of verified users with the specified ID.
         
         Arguments:
             user_id: The ID of the user to be listed.
             search_query: The search query for name of the users to be found.
             is_verified: Status of the user's verification; None when provided as an argument.
+            sort_order: Sort the list of users; 1 for ascending order and -1 for descending order
```

Comment:
Optimize as: 
sort_order as a boolean, that way you can handle ascending, descending and no sort.
I'll leave the implementation for you

Reply:
How about we use this naming convention? [link here!](https://www.django-rest-framework.org/api-guide/filtering/#orderingfilter)